ui-tests/cypress/support/pageObjects/content/management/AddPage.js

```diff
@@ -176,6 +186,10 @@ export default class AddPage extends Content {
     return this;
   }
 
+  getOpenedDialog() {
```

Comment:
You should already have access to the dialog through the AppPage object, which should contain this AddPage as Content; please validate it and update the test accordingly

Reply:
done